feat: Document how to use the HTTP client override parameter (#530)
serverpod/serverpod_docs: docs/06-concepts/01-working-with-endpoints/01-working-with-endpoints.md

  publicScheme: http
```

> See [Configure HTTP calls](./working-with-endpoints/configure-http-calls) for transport-specific configuration such as browser credentials or platform-native HTTP clients.

Reviewer: Nit: minor clarity improvement.

```suggestion
To enable browser credentials for CORS or use platform-native HTTP clients, see [Configure HTTP calls](./working-with-endpoints/configure-http-calls).
```
